uart_sssss.c

```diff
@@ -54,3 +54,29 @@ void uart2_transmit(int ch)
 ///
 ///
 
+
+void pa_0_EXTI_Init(void)
+{
+	/*
+//Disable global interrupt
+	__disable_irq();
+//Enable clock access for GPIOA
+RCC->AHB1ENR |= (1U << 0);
```

Comment:
set some macros to the bits in order to made them easy to read without the need of the datasheet to be read

Reply:
done